assignments/prototypes.js

```diff
@@ -15,14 +15,36 @@
   * dimensions (These represent the character's size in the video game)
   * destroy() // prototype method that returns: `${this.name} was removed from the game.`
 */
-
+//GameObject Constructor 
+function GameObject(attrs) {
+  this.createdAt = attrs.createdAt;
+  this.name = attrs.name;
+  this.dimensions = attrs.dimensions;
+}
+//Prototype method
+GameObject.prototype.destroy = function() {
+  return `${this.name} was removed from the game.`;
+};
 /*
   === CharacterStats ===
   * healthPoints
   * takeDamage() // prototype method -> returns the string '<object name> took damage.'
   * should inherit destroy() from GameObject's prototype
 */
+//Constructor 
+function CharacterStats(attrs) {
+  this.createdAt = attrs.createdAt;
+  this.name = attrs.name;
+  this.dimensions = attrs.dimensions;
+  this.healthPoints = attrs.healthPoints;
```

Comment:
we could use `GameObject.call(this, attrs);` to bind `this` to GameObject and get all three properties.